contracts/AssetTransfer.sol

```diff
@@ -36,6 +15,20 @@ contract AssetTransfer is WorkbenchBase('AssetTransfer', 'AssetTransfer')
         Terminated
     }
 
+    event ContractCreated(string applicationName, string workflowName, address originatingAddress);
+    event ContractUpdated(string applicationName, string workflowName, string action, address originatingAddress);
+
+    string internal ApplicationName;
+    string internal WorkflowName;
+
+    function LogContractCreated() private {
```

Comment:
I know the original contract had these functions in the abstract contract for workbench.  Thinking it might make sense to remove the function and just emit the events from the calling function.

Reply:
Good point. I also thought about this problem. If the private functions do not reduce much code duplication, I think it is a good idea to remove them